dborutils/key_service.py

```diff
@@ -3,6 +3,12 @@
 from numpy import base_repr
 
 
+# Changing this will require DB migrations in both dbor_wip and
+# noodle-api!!!  Do not reduce below the longest existing nice_keys
+# across all tables!!!
+NICE_KEY_MAX_LENGTH = 20
```

Comment:
The target length at assignment time will change to 12 (that happened in a previous PR), but NICE_KEY_MAX_LENGTH refers to the database column size which obviously must be the same or larger.  I don't know if there are any existing keys that are longer than 12, but the collision-detection algorithm might bump up to as long as I think 15 if it fires (which is now very unlikely).  20 is what I set everything to in DBOR when I standardized it a while ago. noodle-api is currently everywhere from 10 to 255, so this will bring it into sync with DBOR.